enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer/001_kube_enforcer_config.yaml

```diff
@@ -153,8 +153,8 @@ rules:
 #    resources: ["imagecontentsourcepolicies", "openshiftapiservers", "kubeapiservers"]
 #    verbs: ["get", "list", "watch"]
 #  - apiGroups: [ "" ]
-#    resources: [ "serviceaccounts", "endpoints" ]
-#    verbs: [ "list" ]
+#    resources: [ "serviceaccounts", "endpoints"]
```

Comment:
this needs to be corrected.
also, as discussed remove serviceaccounts from here.

Reply:
afaik serviceaccounts belong to the core spiGroup "" and it's a good practice to keep them associated with "". As discused , moving it to "*"(Line 132 above) broadens the allowed scope - and is generally not good practice. That's my concern. If you still see reasons to shift it above, please suggest. Thanks for the review.